etc/bento_dev.env

```diff
@@ -14,6 +14,7 @@ BENTO_BEACON_NETWORK_ENABLED='false'
 BENTO_CBIOPORTAL_ENABLED='false'
 BENTO_GOHAN_ENABLED='true'
 BENTO_MONITORING_ENABLED='false'
+BENTO_MINIO_ENABLED='true'
```

Comment:
I think we should stick to 'false' by default, at least for now. You can enable your Minio service by overriding this variable in `local.env`

Reply:
@v-rocheleau Done.